src/backend/distributed/sql/udfs/create_distributed_table/latest.sql

```diff
@@ -0,0 +1,15 @@
+DROP FUNCTION create_distributed_table(regclass, text,  citus.distribution_type, text);
+CREATE OR REPLACE FUNCTION create_distributed_table(table_name regclass,
+                                                    distribution_column text,
+                                                    distribution_type citus.distribution_type DEFAULT 'hash',
+                                                    colocate_with text DEFAULT 'default',
+                                                    shard_count int DEFAULT current_setting('citus.shard_count')::int)
```

Comment:
Do we need to have some logic around colocate_with + shard_count?

I'm wondering whether the default should actually be NULL, which would mean:
- If auto-colocatable, disallow shard_count
- If colocate_with := '<table>', disallow shard_count
- If colocate_with := 'none' or first table in co-location group, use shard_count argument, or fall back to citus.shard_count if NULL